### PR TITLE
[CORE-1981] Add tests and fix for zero CPU values

### PR DESCRIPTION
**Pull request by robert-uhl**
And fix zero CPU in pachctl input.

### CODE REVIEW

File: src/server/pps/cmds/defaults_test.go [hunk 1]
```diff
@@ -249,3 +249,61 @@ func TestCreatePipeline_delete(t *testing.T) {
 		"RepoName", "input",
 		"PipelineName", "pipeline").Run())
 }
+
+func TestCreatePipeline_leading_zero(t *testing.T) {
```
Comment (acohen4) [inline src/server/pps/cmds/defaults_test.go:253]: what do you think of just having a unit test for yamlToJson instead of this integration test?

Comment (robert-uhl) [reply src/server/pps/cmds/defaults_test.go:253]: I implemented this test as a regression test for the user-experienced bug. But you’re right that a unit test for the function itself would be good … I’ll add one.

**Review by acohen4 — APPROVED** [review]
LGTM. Having an integration test for pachctl + gRPC seems redundant to me. Though it's much better to over test than under test
